enzy_htp/structure/structure.py

```diff
@@ -291,7 +291,8 @@ def find_atoms_in_range(self, center: Union[Atom, Tuple[float, float, float]], r
         return result
 
     def find_idx_atom(self, atom_idx: int) -> Atom:
-        """find atom base on its idx. return a reference of the atom."""
+        """find atom base on its idx. return a reference of the atom.
+        NOT! suitable for heavy duty jobs. pretty slow. use atom_idx_mapper instead"""
```

Comment:
You learned about this one the hard way lol.

Reply:
lolllllllllll fr